helm/openwhisk/runtimes.json

```diff
@@ -175,6 +175,22 @@
                 }
             }
         ],
+        "rust": [
+            {
+                "kind": "rust:1.34",
+                "default": true,
+                "deprecated": false,
+                "attached": {
+                    "attachmentName": "codefile",
+                    "attachmentType": "text/plain"
+                },
+                "image": {
+                    "prefix": "openwhisk",
+                    "name": "actionloop-rust-v1.34",
+                    "tag": "1.13.0-incubating"
```

Comment:
This tag doesn't exist (because we never made an official ASF release of the rust runtime).

Should we be making a release of the rust runtime first?